lib/recurly/external_subscription.rb

```diff
@@ -0,0 +1,17 @@
+module Recurly
+  class ExternalSubscription < Resource
+
+    define_attribute_methods %w(
+      href
+    )
+
+    def href
+      @href
+    end
```

Comment:
We don't typically expose the href of a Resource as an attribute method, instead allowing the links to build associations.